src/runner/browser-job.js

```diff
@@ -85,7 +86,8 @@ export default class BrowserJob extends EventEmitter {
 
     _createTestRun (test, testIndex, quarantineAttemptNum) {
         var TestRunCtor        = test.isLegacy ? LegacyTestRun : TestRun;
-        var screenshotCapturer = this.screenshots.createCapturerFor(test, testIndex, quarantineAttemptNum, this.browserConnection);
+        var screenshotCapturer = this.screenshots.createCapturerFor(test, testIndex,
+            this.opts.quarantineMode ? quarantineAttemptNum : null, this.browserConnection);
```

Comment:
Let's keep it single line, just refactor `this.opts.quarantineMode ? quarantineAttemptNum : null` into var